installer/templates/phx_single/lib/app_name_web.ex

```diff
@@ -64,7 +64,8 @@ defmodule <%= web_namespace %> do
 
       import Plug.Conn
       import Phoenix.Controller<%= if live do %>
-      import Phoenix.LiveView.Router<% end %>
+      import Phoenix.LiveView.Router<% end %><%= if dashboard do %>
+      import Phoenix.LiveDashboard.Router<% end %>
```

Comment:
For discussion: maybe we should import it in the router itself inside the `if Mix.env == :dev` block?

Reply:
It feels less foot-gun'y to leave it here?

Reply:
Having it together helps people understand where “live_dashboard” comes from though and where to look for more info.

Reply:
But I am happy either way!